packages/atomic/src/components/common/atomic-modal/atomic-modal.tsx

```diff
@@ -89,6 +89,13 @@ export class AtomicModal implements InitializableComponent<AnyBindings> {
     }
   }
 
+  @Listen('keyup', {target: 'body'})
+  handleCloseOnEscape(e: KeyboardEvent) {
+    if (e.key.toLowerCase() === 'escape') {
```

Comment:
🤔 why lowerCase and not just "Escape" ?
https://developer.mozilla.org/en-US/docs/Web/API/UI_Events/Keyboard_event_key_values

Reply:
Out of habit after all these years doing web dev, I always lower case compare anything that's coming from DOM API due to not knowing by heart where are all the landmines.